core/src/main/java/hudson/DNSMultiCast.java

```diff
@@ -1,205 +1,22 @@
 package hudson;
 
 import jenkins.util.SystemProperties;
-import jenkins.model.Jenkins;
-import jenkins.model.Jenkins.MasterComputer;
 
-import javax.jmdns.ServiceInfo;
-import javax.jmdns.impl.JmDNSImpl;
 import java.io.Closeable;
 import java.io.IOException;
-import java.lang.reflect.Field;
-import java.lang.reflect.InvocationTargetException;
-import java.lang.reflect.Method;
-import java.net.InetAddress;
-import java.net.URL;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.concurrent.Callable;
-import java.util.concurrent.ExecutorService;
-import java.util.logging.Level;
-import java.util.logging.Logger;
 
 /**
  * Registers a DNS multi-cast service-discovery support.
  *
  * @author Kohsuke Kawaguchi
+ * @deprecated No longer does anything.
  */
+@Deprecated
```

Comment:
Add Restricted(DoNotUse) to fail builds?

Reply:
I couldn't get it to work without failing the Jenkins build. We could use NoExternalUse, though.

Reply:
https://github.com/kohsuke/access-modifier/pull/17